src/models/accounts.js

```diff
@@ -44,19 +44,25 @@ function * getAccounts () {
 function * getAccount (name, requestingUser) {
   log.debug('fetching account name ' + name)
 
-  const canExamine = requestingUser &&
-    (requestingUser.name === name || requestingUser.is_admin)
-  const account = yield db.getAccount(name)
-  if (!account) {
-    throw new NotFoundError('Unknown account')
-  } else if (account.is_disabled &&
+  let canExamine = false
+  let account
+  if (!requestingUser) { // anonymous request
+    account = {id: name, name: name}
```

Comment:
This object is not the object that is returned in the HTTP response. Once it has [been converted](https://github.com/interledgerjs/five-bells-ledger/blob/f4055ea73bad19211c96a96d0619acce93fc4d66/src/models/accounts.js#L67), `id` is an URL.